spinel-cli.py

```diff
@@ -790,6 +790,10 @@ def do_counters(self, line):
                 TxBeaconRequest: 0
                 TxOther: 0
                 TxRetry: 0
+                    mTxDirectRetrySuccess: [ 1:2, 2:2, 3:1 ]
```

Comment:
```suggestion
                    TxDirectRetrySuccess: [ 0:2, 1:2, 2:1 ]
```

I would the character "m".